src/containers/account/components/TransferList.js

```diff
@@ -152,7 +152,7 @@ class TransferList extends React.Component {
                         </StyleTable>
                     </div>
                 )}
-                {transfers && !this.isLastPage() && (
+                {transfers && transfers.length > 0 && !this.isLastPage() && (
```

Comment:
this breaks account history. if you had no transactions in the past 30 days (ie. the first loaded batch), the button is not shown thus there's no way to see your history.